docs/appendix.rst

```diff
@@ -323,6 +323,94 @@ This documentation uses a few special terms to refer to Python types:
       collection. Therefore, a caller must be careful to not accidentally
       modify the provided :class:`~pywbem.CIMQualifier` objects.
 
+
+.. _`Profile advertisement methodologies`:
+
+Profile advertisement methodologies
+-----------------------------------
+
+This section briefly explains the profile advertisement methodologies defined
+by DMTF. A full description can be found in :term:`DSP1033`.
+
+These methodologies describe how a client can discover the central instances
+of an implementation of a management profile. Discovering the central instances
+through a management profile is the recommended approach for clients, over
+simply enumerating a CIM class of choice. The reason is that this approach
+enables clients to work seamlessly with different server implementations, even
+when they have implemented a different set of management profiles.
+
+DMTF defines three profile advertisement methodologies in :term:`DSP1033`:
+
+* GetCentralInstances methodology (new in :term:`DSP1033` 1.1)
+* Central class methodology
+* Scoping class methodology
+
+At this point, the GetCentralInstances methodology has not widely been
+implemented, but pywbem supports it nevertheless.
+
+All three profile advertisement methodologies start from the
+`CIM_RegisteredProfile` instance representing the implementation of the
```

Comment:
change the word implementation to "definition of a particular defined profile including organization, name, and version"

Reply:
The RP instance does not represent the profile document (i.e.l the definition), but the profile implementation:
Two implementations -> Two sets of central instances -> Two RP instances required
We discussed this in the last call with Mike.

Reply:
Does it not represent both, a) the definition of the org/name/version of the profile and the link to the implementation.

Reply:
What do you mean with "definition"? There is definitely no definition of the profile in the sense that CIM instances would represent the profile specification, i.e. like the content of a MRP variant of the profile just in CIM. For example, there is not any representation of a requirement level of any profile element.

I would say the full version of the statement would be that the RP instance represents an implementation of the profile identified by its org/name/version. I can add "identified by its org/name/version" if that helps.